src/main/resources/com/google/api/codegen/java/package-info.snip

```diff
@@ -0,0 +1,58 @@
+@extends "common.snip"
+@extends "main.snip"
+
+@override generateFilename(service)
+  package-info.java
+@end
+
+@override generateClass(service, body, imports)
+  {@fileHeader(service)}
+  
+  {@autoGenServiceWarning()}
+  
+  {@packageJavaDoc()}
+  
+  package {@context.getApiConfig.getPackageName};
+@end
+
+@override generateBody(service)
```

Comment:
It looks like you are contorting this snippet to fit the existing JavaSnippetSet interface, which I don't really feel is necessary.